### PR TITLE
feat(Event decorator): Changed Event decorator to take two arguments

### PR DESCRIPTION
**Pull request by xolir**
BREAKING CHANGE: Current usage of Event decorator will throw errors

### CODE REVIEW

**Comment by mateuszluczak** [comment]
@xolir we will merge this soon, can you resolve conflicts?

File: src/decorators/event.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import handleError, { warn } from '../util/error';
  * @param {string} event
  * @returns (Function} decorator
  */
-export default function decorator(event, preventDefault) {
+export default function decorator(event, selector, preventDefault) {
```
Comment (mateuszluczak) [inline src/decorators/event.js]: @xolir 
Silly thought about `preventDefault`. Something that was possible before and now is not:

```
@Evt('click', true)
```

You would need to do:
```
@Evt(`click`,,true)
```
But it's both ugly and won't work.

What is your suggested approach here?

Will you change the decorator to be more smart about second decorator being a boolean, or maybe we can slim down the API and get of rid of that option (I don't know the actual usage in projects - you can verify that).

File: examples/dynamic/index.html [hunk 2]
```diff
@@ -29,13 +29,15 @@
 
     @Strudel.Component('.count-control')
     class CountControl {
-      @Strudel.Evt('click .control-increment')
+      @Strudel.Evt('click', '.control-increment')
       increment() {
+        debugger;
```
Comment (mateuszluczak) [inline examples/dynamic/index.html]: Remove this

File: examples/dynamic/index.html [hunk 3]
```diff
@@ -29,13 +29,15 @@
...
         this.$emit('counter.increment');
       }
 
-      @Strudel.Evt('click .control-decrement')
+      @Strudel.Evt('click', '.control-decrement')
       decrement() {
+        debugger;
```
Comment (mateuszluczak) [inline examples/dynamic/index.html]: Same here

File: src/decorators/event.js (hunk 1 above)
Comment (mateuszluczak) [reply src/decorators/event.js]: Screenshot 2019-06-12 at 09 54 07

Comment (mateuszluczak) [reply src/decorators/event.js]: Screenshot 2019-06-12 at 16 00 05

**Comment by xolir** [comment]
Closing this in favour of https://github.com/strudeljs/strudel/pull/101

**Pull request closed without merging**